Added basic glade example

drakbar opened the pull request.

drakbar committed:
Added basic glade example

Bios-Marcel commented:
We might not want to encourage manual invocation of `gtk.Init` and `gtk.Main`. Mind switching it to use `Application.Run` and connect the startup stuff to the `activate` signal? I should at some point fix that throughout the examples as well.

drakbar committed:
Update glade.go

Refactored to use GtkApplication as per https://wiki.gnome.org/HowDoI/GtkApplication.

drakbar committed:
Update example.glade

Changed the id of the GtkWindow and the name of the signal handler since they should be unique.

drakbar commented:
I agree, and have made the requested changes.

Bios-Marcel commented:
Already merged ;)
